tree_unique_args/tests/interpreter.rs

```diff
@@ -257,6 +301,7 @@ pub fn interpret(e: &Expr, arg: &Option<Value>, vm: &mut VirtualMachine) -> Valu
             let Some(v) = arg else { panic!("arg") };
             v.clone()
         }
+        Expr::Function(_, _) | Expr::Call(_, _) => todo!("interpret functions and calls"),
```

Comment:
Another good issue to create. I think what would be best is to make a top level `(Program (Cons func_1 ...))` that we could interpret with some inputs.